Source/Charts/Components/YAxis.swift

```diff
@@ -150,6 +150,23 @@ open class YAxis: AxisBase
         var min = _customAxisMin ? _axisMinimum : dataMin
         var max = _customAxisMax ? _axisMaximum : dataMax
         
+        // Make sure max is greater than min
+        if min > max
+        {
+            if _customAxisMax && _customAxisMin
+            {
+                (min, max) = (max, min)
+            }
+            else if _customAxisMax && !_customAxisMin
+            {
+                min = max - 1
```

Comment:
what you think if we use `min = 0.5 * max` and `max = 1.5 * min`? @potato04 @jjatie @petester42 ?
clearly if min > max, and user only set one custom value of both, +/- 1 is not always the good value, for example with large values and decimals

Reply:
Couldn't agree more, may be we should update those codes too 
https://github.com/danielgindi/Charts/blob/a77a358fb0d8a89b6121d4cdcbc122cc42cce395/Source/Charts/Components/YAxis.swift#L170-L178

Reply:
When I was adding unit tests, I found that the correction for the maximum and minimum values were wrong.
Because we just modified `min` and `max` instead of `_axisMinimum` and `_axisMaximum`.
I think we should do that in the setter function of `axisMinimum`/`axisMaximum` if it is necessary.
https://github.com/danielgindi/Charts/blob/86ae25384960e764641650bd76ef20fb9148aa96/Source/Charts/Components/AxisBase.swift#L307-L319
@liuxuan30